cookbooks/fb_apt/recipes/default.rb

```diff
@@ -64,7 +64,18 @@
   mode '0644'
 end
 
+execute 'check keyring format' do
```

Comment:
This resource name isn't great, given that this may mutate and 'check' makes me think otherwise.